### PR TITLE
Fix and update aklite-offline return codes based on v93 behavior

### PR DESCRIPTION
**Pull request by detsch**
# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [ x Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 66ed1a7a7911b2da5acd4f8078d8ca9ade1b5e63 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2642/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
LGTM

File: source/user-guide/offline-update/offline-update.rst [hunk 1]
```diff
@@ -146,13 +146,13 @@ The ``run`` command sets the following exit codes:
...
 - *99*: Update failed.
     - Device failed to boot on the updated rootfs and rolled back to the previous version.
-- *100*: Update failed.
-    - Device successfully booted on the updated rootfs but failed to start the updated Apps after the reboot.
-    - Device is rolling back to the previous version, reboot followed by ``aklite-offline run`` is required to complete the rollback.
 - *110*: Update failed.
     - Device failed to boot on the updated rootfs and rolled back to the previous version.
     - Device failed to start the previous version's Apps since they are unknown.
-- *120*: Update and rollback failed.
+- *120*: Update failed.
```
Comment (mike-sul) [inline source/user-guide/offline-update/offline-update.rst:158]: Actually the `aklite-offline run` returns `100` in this case, please see this code https://github.com/foundriesio/aktualizr-lite/blob/4733c7f7980c0874ad603c20e6d16ec834793753/apps/aklite-offline/cmds.cpp#L90.

I did so to preserve the previous version returns code.

File: source/user-guide/offline-update/offline-update.rst [hunk 2]
```diff
@@ -146,13 +146,13 @@ The ``run`` command sets the following exit codes:
...
+    - Device successfully booted on the updated rootfs but failed to start the updated Apps after the reboot.
+    - Device is rolling back to the previous version, reboot followed by ``aklite-offline run`` is required to complete the rollback.
+- *130*: Update and rollback failed.
```
Comment (mike-sul) [inline source/user-guide/offline-update/offline-update.rst:161]: The same here, `120` is returned https://github.com/foundriesio/aktualizr-lite/blob/4733c7f7980c0874ad603c20e6d16ec834793753/apps/aklite-offline/cmds.cpp#L98.

**Comment by detsch** [comment]
I've dropped the first patch, which was not correct as reported by Mike Sul.

**Comment by doanac** [comment]
Docs for 8c354439f9fd6a7aa84c07ba8fd88e16c391fd74 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2643/docs/artifacts/html/index.html

**Pull request merged**